docs(testing): fix wrong documentation for fastify e2e tests (#1766)
nestjs/docs.nestjs.com: content/fundamentals/unit-testing.md

});
```

> > info **Hint** If you're using [Fastify](/techniques/performance) as your HTTP adapter, it requires slightly different configuration, and has build-in testing capabilities:

Reviewer: ```suggestion
> info **Hint** If you're using [Fastify](/techniques/performance) as your HTTP adapter, it requires a slightly different configuration, and has built-in testing capabilities:
```

>
> ```ts
> let app: NestFastifyApplication;
> 
> beforeAll(async () => {
>   app = moduleRef.createNestApplication<NestFastifyApplication>(
>     new FastifyAdapter(),
>   );
>
>   await app.init();
>   await app.getHttpAdapter().getInstance().ready();
> })
> 
> it(`/GET cats`, () => {
>   return app
>     .inject({
>       method: 'GET',
>       url: '/cats'
>     }).then(result => {

> >       expect(result.statusCode).to.be.eql(200)

Reviewer: ```suggestion
>       expect(result.statusCode).toEqual(200)
```

> >       expect(result.payload).to.be.eql('up')

Reviewer: ```suggestion
>       expect(result.payload).to.be.eql(/* expectedPayload */)
```

> >       expect(result.payload).to.be.eql(/* expectedPayload */)

Reviewer: ```suggestion
>       expect(result.payload).toEqual(/* expectedPayload */)
```
